package rec.slow;

message GuiSettings {
  optional bool show_grid = 1 [default = true];
  optional bool drop_adds_to_browser = 2 [default = true];
  optional bool parallel_waveforms = 3 [default = false];
  optional bool show_names_in_waveform = 4 [default = true];
};
